troubleshooting/README.md

```diff
@@ -57,6 +57,7 @@ to the exiting problem seen into the existing search. (The example searches for
 * [VCS Password With Illegal Characters](known_issues/VCS_Password_With_Illegal_Characters.md)
 * [BOS Sessions Stuck Pending](known_issues/BOS_Sessions_Stuck_Pending.md)
 * [BOS Operator Pods `OOMKilled`](known_issues/BOS_Operator_Pods_OOMKilled.md)
+* [Soft Deleted IMS recipe always has `require_dkms=true`](known_issues/ims_soft_deleted_recipe_always_has_require_dkms_true.md)
```

Comment:
Top-level headers for pages (and links to those pages) should capitalize all words except ones like `a`, `the`, etc, and except for ones that are literal strings, like you have here with `require_dkms=true`
```suggestion
* [Soft Deleted IMS Recipe Always Has `require_dkms=true`](known_issues/ims_soft_deleted_recipe_always_has_require_dkms_true.md)
```